src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -560,6 +564,31 @@ def getSystemParameters(self):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def switchLed(self, switchTo):
```

Comment:
Instead of "switchTo" parameter better "enabledState" or something

Reply:
According to my new suggested naming of the `readImage()` method parameter, i would propose the method with following signature: `switchLedManually(enabledState)`